packages/datagateway-common/src/state/reducers/dgcommon.reducer.tsx

```diff
@@ -196,6 +341,19 @@ export function handleClearTable(state: DGCommonState): DGCommonState {
     error: null,
     sort: {},
     filters: {},
+    savedView: {
+      view: null,
+      queries: null,
+      filters: {},
+      sort: {},
+    },
```

Comment:
`filterData` and `query` are only used in card view, so it make senses not to include in `clearTable`. The `query` gets preserved by the `savedView` object and `filterData` is distinct from `filters` so isn't affected by table view.
One thing I did notice is that filtering/sorting that is a shared option between card/table is preserved for each view independently. For example we can sort the table ascending, and the cards descending which will then remain in place when swapping back and forth. This seems counter intuitive to me but might be intended?

Reply:
@patrick-austin . at the time I think I did consider this briefly but I think the idea was whatever queries one view would have, it would belong with that view. It would make sense for the same queries to be used on either view since you are only alternating between views and not different information, but I think the issue at that time was whether the queries would work on both views. 

At the moment, like you said, the table view and card view queries are separate but as long as the table view queries do work completely alongside the card view, the views do not need to be swapped.